src/main/resources/bootstrap.yaml

```diff
@@ -5,4 +5,3 @@ spring:
       prefixed: false
       paths: /mnt/secrets/reform-scan
       aliases:
-        app-insights-instrumentation-key: azure.application-insights.instrumentation-key
```

Comment:
this can stay here ?

Reply:
but this one perhaps i need to `enabled: false`. dunno, pr is green and i would guess it is using bootstrap setup ^

Reply:
you can keep this although app insights key will be still be the old one. Once you have correct value in key vault it will set automatically.

Reply:
i mean the vault itself will be deleted so there won't be any value to load from